hazelcast/src/main/java/com/hazelcast/core/Member.java

```diff
@@ -40,6 +41,14 @@
     boolean localMember();
 
     /**
+     * Returns the Address of this Member.
+     *
+     * @return the address.
+     * @since 3.6
```

Comment:
BTW, this change makes `Address` a public API by exposing it here. Before it was an internal thing.